README.md

```diff
@@ -0,0 +1,12 @@
+# Learning on a foreign project
+I was interested in Rust and cloned a Project, that was not maintained. There was no documentation of
+what the program should do or if it ever did something. When I started, it was failing at Compilation.
```

Comment:
Not sure these lines are relevant for the project: it makes sense in your fork, but would not be clear in this repository. What is your opinion?